### ISSUE TITLE
[no-duplicate-attrs] Bug - the attribute '' is duplicated

### ISSUE THREAD

**Issue by gian1200** [yeonjuan/html-eslint#110]
## Description
After upgrading the dependencies it wrongfully started flagging the code.
When removing the newline it worked as expected.
It also fails when running ESLint via CLI.

## Upgrade path
From:
- "@html-eslint/eslint-plugin": "^0.13.2"
- "@html-eslint/parser": "^0.13.2"

To:
- "@html-eslint/eslint-plugin": "^0.14.1"
- "@html-eslint/parser": "^0.14.0"

## Error
![image](https://user-images.githubusercontent.com/405347/192338589-5626f185-2357-47dc-b009-178f1b323174.png)

- VSCode extension:
![image](https://user-images.githubusercontent.com/405347/192339413-35a04cc0-5af6-4847-ba88-57634512f064.png)

- Terminal:
![image](https://user-images.githubusercontent.com/405347/192339899-e8fff37d-be8f-4476-a170-4f56d2ba81d9.png)

## No error (when removing the breakline):
![image](https://user-images.githubusercontent.com/405347/192338647-5d66393d-6b2a-4f02-b07b-0bec51d788a5.png)

**Comment by yeonjuan** [comment]
@gian1200  Hi, I can't reproduce this bug. Could you paste the problem code on the comment?

**Comment by gian1200** [comment]
I wrote a new file, same error (I also checked for invisible characters).

```html
<!DOCTYPE html>
<html lang="es">

<head>
	<meta charset="utf-8" />
	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
	<title>Page Title</title>
	<meta name="viewport" content="width=device-width, initial-scale=1" />
	<link rel="stylesheet" type="text/css" media="screen" href="main.css" />

</head>

<body>
	<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none"
		stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"
		class="feather feather-home">
		<path d="M3 9l9-7 9 7v11a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2z"></path>
		<polyline points="9 22 9 12 15 12 15 22"></polyline>
	</svg>
</body>

</html>
```
Just in case I also add eslint.config.js file;

```javascript
/**
 * @type {import("eslint").Linter.Config}
 */
module.exports = {
	"env": {
		"es6": true
	},
	"extends": [
		"eslint:recommended"
	],
	"ignorePatterns": ["!.*", "dist/*"],
	"overrides": [
		{
			"env": {
				"browser": true,
				"jquery": true
			},
			"extends": [
				"plugin:jsdoc/recommended"
			],
			"files": ["src/app/**/*.js"],
			"globals": {
				...
				<removed for privacy>
				...
			}
		},
		{
			"env": {
				"node": true
			},
			"files": [".eslintrc.js", "karma.conf.js", ".mocharc.js", "stylelint.config.js", "webpack.*.js"],
			"parserOptions": {
				"ecmaVersion": 2022
			}
		},
		{
			"env": {
				"node": true
			},
			"extends": [
				"plugin:jsdoc/recommended"
			],
			"files": ["**/*.mjs"],
			"parserOptions": {
				"ecmaVersion": 2022,
				"sourceType": "module"
			}
		},
		{
			"env": {
				"jasmine": true
			},
			"files": ["src/app/**/*.spec.*"],
			"globals": {
				"fetchMock": "readonly"
			},
			"parserOptions": {
				"ecmaVersion": 2022
			}
		},
		{
			"env": {
				"mocha": true
			},
			"files": ["src/server/**/*.spec.*"],
			"parserOptions": {
				"ecmaVersion": 2022
			}
		},
		{
			"extends": [
				"plugin:@html-eslint/recommended"
			],
			"files": ["src/**/*.html"],
			"parser": "@html-eslint/parser",
			"plugins": [
				"@html-eslint"
			],
			"rules": {
				// TODO https://github.com/yeonjuan/html-eslint/issues/101. WARN
				"@html-eslint/indent": ["off", "tab"],
				// TODO https://github.com/yeonjuan/html-eslint/issues/100. WARN
				"@html-eslint/no-extra-spacing-attrs": ["off"],
				"@html-eslint/require-closing-tags": ["error", { "selfClosing": "always" }]
			}
		}
	],
	"plugins": [
		"jsdoc"
	],
	"root": true,
	"rules": {
		"array-bracket-newline": ["error", "consistent"],
		"array-bracket-spacing": ["error", "never"],
		"array-callback-return": ["error"],
		"array-element-newline": ["error", { "ArrayExpression": "consistent" }],
		"arrow-spacing": ["warn", { "before": true, "after": true }],
		"block-spacing": ["warn", "always"],
		"comma-dangle": ["error", "never"],
		"comma-spacing": ["warn", { "before": false, "after": true }],
		"curly": ["error", "all"],
		"default-param-last": ["error"],
		"eqeqeq": ["error", "smart"],
		"func-call-spacing": ["error", "never"],
		"grouped-accessor-pairs": ["error", "getBeforeSet"],
		"indent": ["warn", "tab"],
		"key-spacing": ["warn", { "afterColon": true }],
		"keyword-spacing": ["warn"],
		"linebreak-style": ["off"],
		"lines-around-comment": ["error"],
		"lines-between-class-members": ["error", "always"],
		"no-console": ["warn"],
		"no-debugger": ["warn"],
		"no-empty": ["warn"],
		"no-lonely-if": ["warn"],
		"no-multi-spaces": ["error"],
		"no-multiple-empty-lines": ["warn", { "max": 1, "maxEOF": 0 }],
		"no-multi-str": ["error"],
		"no-return-assign": ["error"],
		"no-shadow": ["warn", { "builtinGlobals": true, "hoist": "all" }],
		"no-trailing-spaces": ["warn"],
		"no-unneeded-ternary": ["error"],
		"no-unused-vars": ["warn"],
		"no-var": ["error"],
		"nonblock-statement-body-position": ["error", "below"],
		"object-curly-newline": ["error", { "consistent": true }],
		"object-curly-spacing": ["warn", "always"],
		"object-property-newline": ["error", { "allowAllPropertiesOnSameLine": true }],
		"padded-blocks": ["warn", { "blocks": "never", "classes": "always", "switches": "never" }, { "allowSingleLineBlocks": false }],
		"prefer-const": ["warn"],
		"quotes": ["warn", "double", { "allowTemplateLiterals": true, "avoidEscape": true }],
		"semi": ["warn", "always"],
		"space-before-blocks": ["warn"],
		"space-infix-ops": ["warn"],
		"space-in-parens": ["warn", "never"]
	}
};
```

**Comment by yeonjuan** [comment]
@gian1200 Thanks for the more information. There is a problem if the code is written with the CRLF (window's newline). I'm going to fixing it

**Comment by yeonjuan** [comment]
Hi, @gian1200 It's fixed in `v10.15.0`. I suggest update the versions both `@html-eslint/parser and` `@html-eslint/eslint-plugin`. 
Thanks!

```bash
npm install -D @html-eslint/eslint-plugin@^0.15.0 @html-eslint/parser@^0.15.0
```
